assets/template/Dockerfile

```
@@ -0,0 +1,12 @@
+# This file is used for building the container, ensuring installation of the required tool and
+# dependencies
+
+# If there is any dedicated image available, you should build the benchmarking image on top of that
+FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
+
+# Install necessary packages
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)